src/query.jl

```diff
@@ -177,8 +178,8 @@ function partial_signature_matching(f::Function, sig::Tuple)
 end
 
 """
-Search loaded documentation for all methods of a generic function `f` that match
-the provided signature `sig`. Optionally, provide an index (1-based) to view an
+Search for all methods of a generic function `f` that match the signature `sig`.
+Optionally, provide an index (1-based) to view an
```

Comment:
Can I do it like this:

```
Search loaded documentation for all methods of a generic function `f` 
that match the provided signature `sig`.
```
So that for the index describtion the first line is not too long - or better the whole sentence in the first line?